src/components/NavHeaderWithButton.js

```diff
@@ -72,7 +72,7 @@ const styles = StyleSheet.create({
         borderColor: Theme.palette.borderColor,
         borderBottomWidth: Platform.OS === "ios" ? 0 : 1,
         zIndex: 10000,
-        backgroundColor: "white",
+        backgroundColor: "#f3f6f4",
```

Comment:
We should not have hard code color values, see line 72 to use colors defined on Theme. Make sure to replace all other hard coded color values through out this PR